ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__weekly_trigger.yaml

```diff
@@ -0,0 +1,33 @@
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
```

Comment:
@ascerra I think we can remove this `releases` section from here, since we are not provisioning any cluster so there is need to get the latest release of OCP version, WDYT?

Reply:
Thanks for the review, I removed this and re-tested
Here are the results.
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/41649/rehearse-41649-periodic-ci-rhpit-interop-tests-main-weekly_trigger-ocp-self-managed-layered-product-interop/1702034172882718720

I'm not completely sure why at the moment but the job will not run without this value. Its not being used but it also doesn't cause any problems leaving this as it is, where removing it does cause errors.

Reply:
I wonder if we should at least switch it to a stable build rather than nightly. It probably won't cause issues, but best to err on the side of caution.

Reply:
Thank you @ascerra for addressing this and testing the changes, I guess we cannot remove this even if it is not used.